docroot/modules/custom/mass_hierarchy/mass_hierarchy.module

```diff
@@ -123,6 +146,15 @@ function mass_hierarchy_views_post_render(ViewExecutable $view) {
  * Implements hook_form_alter().
  */
 function mass_hierarchy_form_alter(&$form, FormState $form_state, $form_id) {
+
+  if (isset($form['#entity_type']) && $form['#entity_type'] == 'node') {
```

Comment:
Since we are trying to execute this code for the existing nodes I would suggest using hook_node_update

Reply:
I tried that, but that won´t work, I did that but the application started to fail in different places because there was a lock on the table we wanted to modify. I made comments on the ticket about it, and also there must be Circle CI executions that you can see with that information.

Reply:
@beto-aveiga I know we discussed this some last week, but I can't recall the outcome. Is running cron enough to get us the results we want here?

Reply:
@tom-fleming  Yeah, previously, cron wasn't working because I made a typo on the function name.
Now it works as expected. I even wrote a test to ensure that scenario.